docs/UserGuide.adoc

```diff
@@ -275,23 +275,31 @@ Clocks out for the employee John Doe.
 
 ==== Add Order
 
+Important Terminology
+
+* _A customer is anyone who wants to place an order._
+* _A member in the RMS is a returning customer._
```

Comment:
Should be changed to:
A member in RMS is a registered customer who enjoys more benefits than a normal customer.

Reply:
ok that sounds good